lib/ldp.js

```diff
@@ -53,8 +53,8 @@ function LDP(argv) {
   }
 
   ldp.pathFilter = regexp().start(ldp.mount).toRegExp();
-  ldp.xssProxy = argv.xssProxy;
-  ldp.proxyFilter = regexp().start(ldp.xssProxy).toRegExp();
+  ldp.proxy = argv.proxy;
+  ldp.proxyFilter = regexp().start(ldp.proxy).toRegExp();
```

Comment:
Good question, removing that variable. It created a regex based on `ldp.proxy`, however express handles paths as regexp already, so no need for that